swarm/swarm.go

```diff
@@ -241,6 +245,12 @@ func (self *Swarm) APIs() []rpc.API {
 			Public:    true,
 		},
 		{
+			Namespace: "swarmfs",
+			Version:   api.Swarmfs_Version,
+			Service:   api.NewSwarmFS(self.api),
+			Public:    true,
```

Comment:
well I think the idea was that we do not 'public' an API that can read/write the filesystem.
Not sure if there is rpcapis